roles/openshift_prometheus/tasks/main.yaml

```diff
@@ -0,0 +1,26 @@
+---
+
+- name: Create temp directory for doing work in on target
+  command: mktemp -td openshift-prometheus-ansible-XXXXXX
+  register: mktemp
+  changed_when: False
+
+- set_fact:
+    tempdir: "{{ mktemp.stdout }}"
```

Comment:
Should use a more descriptive local name than just `tempdir`. Fix in follow-up.

Reply:
Also going to assume this should be ran on the local control host, not whatever host you run the install playbook against (like `oo_first_master`). But either will work I suppose. 

Check back on this in follow-up PR